Reference HTML forms assessment features not covered in the tutorial (#44013)
mdn/content: files/en-us/learn_web_development/core/structuring_content/html_forms/index.md

#### Specialized text field inputs

The second exercise above raises an interesting point. The second input field specifically expects an email address and validates entered values as such. If you look at the form code again, you'll see why — the second `<input>` has a `type` of `email`.

> There are several specialized text field input types designed to handle specific types of data — [`<input type="color">`](/en-US/docs/Web/HTML/Reference/Elements/input/color), [`<input type="number">`](/en-US/docs/Web/HTML/Reference/Elements/input/number), [`<input type="password">`](/en-US/docs/Web/HTML/Reference/Elements/input/password), [`<input type="tel">`](/en-US/docs/Web/HTML/Reference/Elements/input/tel), [`<input type="url">`](/en-US/docs/Web/HTML/Reference/Elements/input/url), etc.

Reviewer: Perhaps we should drop `<input type="color">` from this list because it is introduced earlier on this page (L199) as an input type that renders a color picker, not a text field.
```suggestion
There are several specialized text field input types designed to handle specific types of data such as [`<input type="number">`](/en-US/docs/Web/HTML/Reference/Elements/input/number), [`<input type="password">`](/en-US/docs/Web/HTML/Reference/Elements/input/password), [`<input type="tel">`](/en-US/docs/Web/HTML/Reference/Elements/input/tel), [`<input type="url">`](/en-US/docs/Web/HTML/Reference/Elements/input/url), and so on.
```

> [!NOTE]
> If you want to have a specific option selected on page load, you can add a `selected` attribute to the relevant `<option>` element.

> [!NOTE]

> > It is possible to split the options inside a `<select>` drop-down menu into multiple subgroups using the {{htmlelement("optgroup")}} element. Have a look at this element's reference page to find out how it works!

Reviewer: ```suggestion
> You can split the options inside a `<select>` drop-down menu into multiple subgroups using the {{htmlelement("optgroup")}} element. Have a look at this element's reference page to find out how it works.
```

Alternatively, could this be regular prose instead of a note, say as a continuation of the para at L463 that says "Each possible value for the data item is..."
So:
"....If you don't specify a `value`, the text inside the `<option></option>` tags is used as the value. You can also split the options inside a `<select>` drop-down menu into multiple subgroups using the {{htmlelement("optgroup")}} element. Take a look at this element's reference page to find out how."

Author: Good call. I've added it as a separate paragraph.

Reviewer: thanks
